community.html

```diff
@@ -82,4 +81,4 @@ <h2 class="alt-h3">Office Hours</h2>
 
 
 
-<script async src="//addtocalendar.com/atc/1.5/atc.min.js" charset="utf-8"></script>
+<script type="text/javascript" src="https://addevent.com/libs/atc/1.6.1/atc.min.js" async defer></script>
```

Comment:
Don't know why I added the `type` 🤔

Reply:
Oh they list it like that here: https://www.addevent.com/add-to-calendar-button

Weirdly has both `async` and `defer` properties on the tag.